com.github.WhatsApp-For-Linux.yml

```diff
@@ -0,0 +1,166 @@
+        sha256: 235a40bec7346c7b82b6a8caae0456353dc06e71f14bc414bcc858af1838719a
+
+  - name: cairomm
+    config-opts:
+      - --disable-documentation
+    cleanup:
+      - /lib/cairomm-1.0
+    sources:
+      - type: archive
+        url: https://www.cairographics.org/releases/cairomm-1.14.3.tar.xz
+        sha256: 0d37e067c5c4ca7808b7ceddabfe1932c5bd2a750ad64fb321e1213536297e78
+
+  - name: glibmm
+    cleanup:
+      - /lib/glibmm-2.4
+    buildsystem: meson
+    config-opts:
+      - -Dbuild-documentation=false
+      - -Dbuild-examples=false
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/glibmm/2.66/glibmm-2.66.2.tar.xz
+        sha256: b2a4cd7b9ae987794cbb5a1becc10cecb65182b9bb841868625d6bbb123edb1d
+
+  - name: pangomm
+    buildsystem: meson
+    config-opts:
+      - -Dbuild-documentation=false
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/pangomm/2.46/pangomm-2.46.1.tar.xz
+        sha256: c885013fe61a4c5117fda395770d507563411c63e49f4a3aced4c9efe34d9975
+
+  - name: atkmm
+    buildsystem: meson
+    config-opts:
+      - -Dbuild-documentation=false
+    cleanup:
+      - /lib/atkmm-1.6
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/atkmm/2.28/atkmm-2.28.2.tar.xz
+        sha256: a0bb49765ceccc293ab2c6735ba100431807d384ffa14c2ebd30e07993fd2fa4
+
+  - name: gtkmm
+    buildsystem: meson
+    config-opts:
+      - -Dbuild-documentation=false
+    cleanup:
+      - /lib/gtkmm-3.0
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/gtkmm/3.24/gtkmm-3.24.5.tar.xz
+        sha256: 856333de86689f6a81c123f2db15d85db9addc438bc3574c36f15736aeae22e6
+
+  - name: ayatana-ido
+    buildsystem: cmake
+    config-opts:
+      - -DCMAKE_INSTALL_PREFIX=/app
+    sources:
+      - type: git
+        url: https://github.com/AyatanaIndicators/ayatana-ido.git
+        tag: 0.9.1
+
+  - name: libayatana-indicator
+    buildsystem: cmake
+    config-opts:
+      - -DCMAKE_INSTALL_PREFIX=/app
+    sources:
+      - type: git
+        url: https://github.com/AyatanaIndicators/libayatana-indicator.git
+        tag: 0.9.1
+
+  - name: intltool
+    buildsystem: autotools
+    config-opts:
+      - --prefix=/app
+    sources:
+      - type: archive
+        url: https://launchpad.net/intltool/trunk/0.51.0/+download/intltool-0.51.0.tar.gz
+        sha256: 67c74d94196b153b774ab9f89b2fa6c6ba79352407037c8c14d5aeb334e959cd
+    cleanup:
+      - '*'
+
+  - name: libdbusmenu-gtk3
+    buildsystem: simple
+    build-options:
+      env:
+        HAVE_VALGRIND_TRUE: '#'
+        HAVE_VALGRIND_FALSE: ''
+    build-commands:
+      - sed -i 's/-Werror//' libdbusmenu-*/Makefile.{am,in}
+      - ./configure --prefix=/app --with-gtk=3 --disable-dumper --disable-static --disable-tests
+      - make
+      - sed -i 's+/usr/lib/x86_64-linux-gnu/girepository-1.0+/app/lib/girepository-1.0+' libdbusmenu-*/Makefile
```

Comment:
I can tell that this will fail on aarch64.

Reply:
Could you please give me some hints what to do about this?